### ISSUE TITLE
Incorrect warning from makecatalogs when icons are in a subdirectory

### ISSUE THREAD

**Issue by hjuutilainen** [munki/munki#764]
Given a pkginfo with:

	<key>icon_name</key>
	<string>testing/munkitools.png</string>

makecatalogs incorrectly outputs a warning about missing icon:

```
$ makecatalogs --force /Volumes/munki/emptyrepo/
Hashing icons/testing/munkitools.png...
Adding pkgsinfo/munkitools-3.0.0.3290.plist to testing...

WARNING: icon_name specified in pkgsinfo/munkitools-3.0.0.3290.plist but it does not exist

Created catalogs/all...
Created catalogs/testing...
```

**Comment by gregneagle** [comment]
Does this happen only for icons in subdirectories?

**Comment by gregneagle** [comment]
And can I assume you are testing with the current Munki 3 beta?

**Comment by gregneagle** [comment]
Should be addressed with https://github.com/munki/munki/commit/dbf85e79c681c67c452fab541d94492ae240a92b

**Comment by hjuutilainen** [comment]
Thanks. Running the latest Munki 3 beta and testing these changes fixed both this and issue #763.
